### PR TITLE
Home health service

### PR DESCRIPTION
**Pull request by hkong**
Very preliminary Home Health Services ruleset for others to work on different sections of it in parallel

### CODE REVIEW

File: HomeHealthServices/R4/files/HomeHealthServicesRule-0.1.0.cql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using FHIR version '4.0.0'
+include FHIRHelpers version '4.0.0' called FHIRHelpers
+
+parameter Patient Patient
+parameter service_request ServiceRequest
+
+define RULE_APPLIES:
+  true
+define PRIORAUTH_REQUIRED:
+  true
```
Comment (yuliumitre) [inline HomeHealthServices/R4/files/HomeHealthServicesRule-0.1.0.cql]: Does HHS need pre-auth? I thought only NEAT needs it.

File: HomeHealthServices/R4/resources/Questionnaire-R4-HomeHealthServices.json [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+          "linkId": "1.1",
+          "text": "Last Name",
+          "type": "string",
+          "required": true,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPatientInfoPrepopulation\".LastName"
```
Comment (yuliumitre) [inline HomeHealthServices/R4/resources/Questionnaire-R4-HomeHealthServices.json]: This is not part of CRD rule. We can keep it in the PR but it should be replaced by the sub-questionnaire (in your DTR PR) :-)

File: HomeHealthServices/R4/files/HomeHealthServicesRule-0.1.0.cql (hunk 1 above)
Comment (hkong) [reply HomeHealthServices/R4/files/HomeHealthServicesRule-0.1.0.cql]: I'll remove it.

**Comment by hkong** [comment]
ah, ok, I'll update

**Comment by lpdecelles** [comment]
Howard, should we be able to test this with CRD?

**Comment by hkong** [comment]
@lpdecelles you need the new pull request I just put into test-ehr, which has a home health service request for William Oster

**Comment by lpdecelles** [comment]
Hello Howard,

This PR has merge conflicts. Could you please fix it?

Thanks,
Larry

**Comment by yuliumitre** [comment]
@hkong @lpdecelles We probably shouldn't merged this PR as it contains the DTR work Howard has been working on and it is WIP. The CRD work has been merged to master via my Home Health Service Plan of Care PR. We could "reject" this PR :-)

**Comment by lpdecelles** [comment]
Howard will create a new PR when DTR is done.

**Pull request closed without merging**